pages/en/about/governance.md

```diff
@@ -12,11 +12,11 @@ The Node.js project follows a [Consensus Seeking][] decision making model.
 ## Collaborators
 
 The [nodejs/node][] core GitHub repository is maintained by the Collaborators
-who are added by the Technical Steering Committee ([TSC][]) on an ongoing basis.
+who are nominated by other existing Collaborators on an ongoing basis.
```

Comment:
That's not what our charter says. Ref https://github.com/nodejs/TSC/blob/main/TSC-Charter.md#section-4-responsibilities-of-the-tsc

Reply:
TSC voting members, as org admin, are the ones who technically add the GH account to the GH team. But definitely the nomination and approval is made by existing collaborators. I prefer Matteo’s suggestion.

Reply:
I don't think the charter says anything about nomination? The only relevant part is:

> the TSC voting members are responsible for all technical development within the Node.js project, including:...
> Maintaining the list of additional Collaborators.

Which isn't contradicting with the descriptions here. The TSC is responsible of maintaining the list, but collaborators can also propose changes to that list. When there are disagreements the TSC is responsible of finding a resolution, that's all.

Also to reference our governance doc https://github.com/nodejs/node/blob/main/GOVERNANCE.md#collaborator-nominations:

> Existing collaborators can nominate someone to become a collaborator.